db/migrations/01_create_peeps_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE peeps(id SERIAL PRIMARY KEY, time_stamp VARCHAR(10), messages VARCHAR(60));
```

Comment:
I like the way you have included a migrations file where all the SQL commands are to make it easier for someone else to use the code

Reply:
Yeah, this is extremely useful